com/paypal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package com.paypal;
+
+public class Perfection {
+
+    private static Perfection perf;
```

Comment:
The others methors are static, there is no necessity to create this instance and the getPerf method, maybe create an private constructor to define this class as a util.